### ISSUE TITLE
the iodide editor doesn't work with screen readers

### ISSUE THREAD

**Issue by Neurrone** [iodide-project/iodide#1629]
Playing with this with the "try it out" link on the site, editing the contents of the text field for source is inaccessible for screen reader users, probably because CodeMirror is being used.

### What I Did

Editing code on https://alpha.iodide.io/tryit/

### What I Expected

Works like a normal editable field

### What Happened 

Unable to hear contents of the field when moving about; probably caused by CodeMirror

### (if applicable) notebook that reproduces bug 

N/A

Also creating this issue in hopes of raising awareness for this. Hoping this becomes a more accessible alternative to Jupyter.

**Comment by bcolloran** [comment]
thanks @Neurrone. We're considering moving to Monaco, which I believe has better behavior around this 
https://github.com/Microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide

**Comment by Neurrone** [comment]
@bcolloran yup it does, the experience it provides in VS Code is pretty good.

**Comment by frastlin** [comment]
I am in support of this move and can't wait until it gets finished!

**Comment by bcolloran** [comment]
Monaco has landed in #2117. i have not checked screen reader support (i don't know how) but i think that means this should be fixed? if not, someone can re-open with additional details
